src/intl/messages.js

```diff
@@ -62,6 +62,13 @@ export const messages: { [messageId: string]: MessageType } = {
   cancel: 'Cancel',
   cannotUpdateCloudInitHostname: 'Host name cannot be synchronized with VM name because the VM name is not a valid host name.',
   cantLogonToConsole: 'Single sign on failed. Please check to see if the guest agent is running on your virtual machine. Contact your administrator if the problem continues.',
+  cantOpenConsole: "Can't open console: {message}",
+  cantOpenGraphicalConsoleEmptyList: "Can't open graphical console since {vmName} is a headless VM.",
+  cantOpenGraphicalConsoleMoreOptions: "Can't open graphical console you can find more options in the VM page.",
```

Comment:
when should this message appear? it's not informative and seems redundant.

Reply:
It's a fallback message with the latest changes is redundant  I removed that :)